src/NzbDrone.Core/Notifications/Telegram/TelegramProxy.cs

```diff
@@ -35,13 +35,13 @@ public TelegramProxy(IHttpClient httpClient, ILocalizationService localizationSe
         public void SendNotification(string title, string message, TelegramSettings settings)
         {
             // Format text to add the title before and bold using markdown
-            var text = $"<b>{HttpUtility.HtmlEncode(title)}</b>\n{HttpUtility.HtmlEncode(message)}";
+            var text = $"*{HttpUtility.HtmlEncode(title)}*\n{HttpUtility.HtmlEncode(message)}";
```

Comment:
If this isn't sending HTML, do we still need to encode it for HTML?

Reply:
For test notifications something like the following inside `Test`:

```
var link = settings.MetadataLink == (int)MetadataLinkType.None ? null : "https://sonarr.tv";

SendNotification(settings.IncludeAppNameInTitle ? brandedTitle : title, body, link, settings);
```

This would allow people to see that the link works.